src/Illuminate/Support/helpers.php

```diff
@@ -559,6 +559,27 @@ function head($array)
     }
 }
 
+if (! function_exists('image_data_uri')) {
+    /**
+     * Convert an image to data URI.
+     * 
+     * @param  string $image
+     * @return string
+     */
+    function image_data_uri($image)
+    {
+        $data_uri = null;
+
+        if (file_exists($image) && ($image_type = exif_imagetype($image))) {
+            $mime_type = image_type_to_mime_type($image_type);
+            $data = base64_encode(file_get_contents($image));
+            $data_uri = "data:$mime_type;base64,$data";
```

Comment:
You can do your `return` directly in the `if()` statement and get rid of the `$data_uri` variable. No return statement is also `NULL`.